server/integration/endpoint/src/test/java/org/infinispan/server/endpoint/EndpointSubsystemOperationsTestCase.java

```diff
@@ -50,48 +46,40 @@ private void executeOp(String operationName, String... parameters) {
       stream(parameters).forEach(cacheNames::add);
       ModelNode result = services.executeOperation(op);
       assertEquals(SUCCESS, result.get(OUTCOME).asString());
+      return result.get(RESULT);
    }
 
    @Test
    public void testIgnoreCaches() throws Exception {
+      assertCachesNotIgnored("cache1", "cache2", "cache3");
+      assertCachesNotIgnored("whatever");
+
       executeOp("ignore-cache-all-endpoints", "cacheA", "cacheB", "cacheC");
       assertCachesIgnored("cacheA", "cacheB", "cacheC");
 
       executeOp("unignore-cache-all-endpoints", "cacheA");
+      assertCachesNotIgnored("cacheA");
```

Comment:
Those seem to be separate test cases... I think it might be a good idea to put them into separate methods.